tests/system/test_misc.c

```diff
@@ -102,7 +102,7 @@ int cmd_sysenter(int argc, char **argv)
    printf("same sleep, but with sysenter:\n");
 
    struct timespec req = { .tv_sec = 0, .tv_nsec = 100*1000*1000 };
-   sysenter_call3(162 /* nanosleep_time32 */, &req, NULL, NULL);
+   sysenter_call3(SYS_nanosleep /* nanosleep_time32 */, &req, NULL, NULL);
```

Comment:
I'm a bit paranoic about replacing 162 with SYS_nanosleep here without an assert like (SYS_nanosleep == 162), simply because I believe I noticed once that even the SYS_* defines can change depending on the libc headers. That's why I intentionally wrote down the actual syscall numbers. It's rare but I believe there was a case where SYS_stat was defined to a different number because there are some many variants of this syscall. The time ones are similar because of the whole effort of eliminating the 32-bit timestamps and... there is some risk that the SYS_nanosleep won't be point to the time32 version.

Reply:
fixed!